src/test/objectstore/test_bluefs.cc

```diff
@@ -679,6 +705,32 @@ TEST(BlueFS, test_compaction_async) {
     writes_done = true;
     join_all(sync_threads);
     fs.compact_log();
+
+    {
+      string dir = "dir.after_compact_test";
```

Comment:
The same problem like before - need to define Dir/File/Data as upper scope const variables

Reply:
done